### PR TITLE
add solution guideline

### PR DESCRIPTION
**Pull request by mairaw**
This came up in a couple of reviews so adding this as an official guideline:
https://github.com/dotnet/samples/pull/641#discussion_r256232487
https://github.com/dotnet/samples/pull/580#discussion_r249222352

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +86,9 @@ To create a sample:
         }
     }
     ```
+    
+5. Don't check in solution files if it contains only one project.
```
Comment (rpetrusha) [inline README.md]:
```suggestion
5. Don't check in the solution file if it contains only one project.
```

**Comment by JohnGoldsmith** [comment]
Hi,  Since I was the offender in #641 I wonder if it would be worth clarifying which route to take as I went straight to CONTRIBUTING.md and that sends you off to:
https://github.com/dotnet/docs/tree/master/CONTRIBUTING.md
...so I didn't read the README properly in the first place.

**Comment by BillWagner** [comment]
@mairaw Is this ready to merge, or are you going to make further updates?

**Comment by mairaw** [comment]
I hadn't merged yet because I wanted to take into account @JohnGoldsmith's feedback first.

**Pull request merged**